src/utils/id-generator.js

```diff
@@ -22,3 +22,12 @@ export function sequentialIdGenerator() {
   let counter = 0;
   return () => String(++counter);
 }
+
+/**
+ * Returns a function that generates a random id in string format.  The random
+ * id will be an integer from 0-99999.
```

Comment:
Nit. You can pass the modules as an argument to that function. That way, the accordion can use it as:

```
const generateRandomId = randomIdGenerator(10000);

...

const [suffix] = useState(generateRandomId);
```

Reply:
Sounds good updated!